.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,26 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
```

Comment:
I stripped this down from the default node workflow which builds on 10.x and 12.x. I'm not really familiar with differences between node versions. Do you think it's worth testing on both?

Reply:
I don't know a ton about what might break against different Node versions, so I'd probably stick with the default

Reply:
👍